Modules/Quest/QuestieQuest.lua

```diff
@@ -1332,6 +1341,7 @@ function QuestieQuest:CalculateAndDrawAvailableQuestsIterative(callback)
                                 --Draw a specific quest through the function
                                 _QuestieQuest:DrawAvailableQuest(quest)
                             else
+                                --* TODO: How the frames are handled needs to be reworked, why are we getting them from _G
```

Comment:
I fully agree. We are totally flooding the global namespace with our stupid frames